CONFIGURATION.md

```diff
@@ -52,6 +52,9 @@ Here are some Stargate-relevant property groups that are necessary for correct s
 | `stargate.jsonapi.operations.lwt.retries`                               | `int` | `3`      | The amount of client side retries in case of a LWT failure.                                                                                                                                         |
 | `stargate.jsonapi.operations.database-config.session-cache-ttl-seconds` | `int` | `300`    | The amount of seconds that the cql session will be kept in memory after last access.                                                                                                                |
 | `stargate.jsonapi.operations.database-config.session-cache-max-size`    | `int` | `50`     | The maximum number of cql sessions that will be kept in memory.                                                                                                                                     |
+| `stargate.jsonapi.operations.default-count-page-size`                   | `int` | `100`    | The default Cassandra page size used for read queries that are used for counting by key purposes.                                                                                                   |
+| `stargate.jsonapi.operations.max-count-limit`                           | `int` | `1000`   | The default max count response when resolved using keys. Set to -1 use cassandra count function.                                                                                                    |
```

Comment:
Maybe reword slightly... "default maximum number of rows to read for count operation" or something? Just to make it clear it limits reads, and not just response (is obvious to us, but someone reading without having background might not be)